spec/models/repository_spec.rb

```diff
@@ -3,4 +3,24 @@
 RSpec.describe Repository, type: :model do
   it { should validate_presence_of :uid }
   it { should validate_uniqueness_of(:uid) }
+
+  describe 'load_repositories' do
+    context 'creates new repo' do
+      let(:organization) { create :organization, login: 'test-organization'  }
+
+      before do
+        Repository.load_repositories(organization)
+      end
+
+      it 'create a new Repository record' do
+        organization.repositories.each do |repository|
```

Comment:
Will pass even if no repositories are created. How does it check that `load_repositories` creates anything (which is explicitly said in the test title)?